marklogic-data-hub-central/ui/src/components/entities/matching/expandable-table-view/expandable-table-view.tsx

```diff
@@ -29,7 +32,7 @@ const testMatchedUriTableColumns = [
     key: "matchedRulesetType " + (counter++) + " exact",
     width: "6%",
     render: (matchedRulesetType, key, index) => (matchedRulesetType.map(rulesetType => {
-      return (rulesetType === "Exact") && <span className={styles.testMatchedColumns} key={key} aria-label={matchedRulesetType + " " + (index)}>
+      return (rulesetType === "exact" || rulesetType === "Exact") && <span className={styles.testMatchedColumns} key={key} aria-label={matchedRulesetType + " " + (index)}>
```

Comment:
If the rulesetType is going to be checked for different variations of the same word then might as well convert it to upper or lower case and check it only once.
something like **rulesetType.toLowerCase() === "exact"**
Same goes for all other places where rulesetType is being checked.